src/frontend/web_application/webpack/config.js

```diff
@@ -18,7 +16,7 @@ const configureStylesheet = () => {
       }),
       // FIXME: useful for deduplication, we need a way prevent that.
       // disabled for dev for build time (10sec instead of 30+)
-      ...(!isDev ? [new OptimizeCssAssetsPlugin({ canPrint: false })] : []),
+      new OptimizeCssAssetsPlugin({ canPrint: false }),
```

Comment:
It is no more disabled for dev build, or is it ? What about deduplication ?

Reply:
it is no more useful thanks to the fix of bootstrap foundation:

previously each time we required it (for flex-grid for example) all the foundation classes were duplicated and so with optimize plugin, the build time was very long and without the plugin the build time was just long.

now with or without, it is almost the same

But without I had some strange behaviors (width of important slider)

Reply:
oh. I forgot to remove the comment